src/lib/core/scroll/scroll.ts

```diff
@@ -0,0 +1,59 @@
+import {Injectable} from '@angular/core';
+import {Scrollable} from './scrollable';
+import {Subject} from 'rxjs/Subject';
+import {Observable} from 'rxjs/Observable';
+import {Subscription} from 'rxjs/Subscription';
+
+
+/**
+ * Service contained all registered Scrollable references and emits an event when any one of the
+ * Scrollable references emit a scrolled event.
+ */
+@Injectable()
+export class Scroll {
```

Comment:
I would call this `ScrollMonitor` or `ScrollDispatcher` (I'm partial to the latter)